README.md

```diff
@@ -49,6 +49,7 @@ This README contains some high level overview information on Atomic App. The det
 8. [Specification coverage](docs/spec_coverage.md)
```

Comment:
@dustymabe considering that the Nulecule format in Atomic App is _still_ different than the format in the spec, I would say to keep it separate. For example, external files for docker source'd applications (ex. the mariadb) is specified in the spec, but we do not support it yet in atomic app